tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -772,6 +772,10 @@ struct Query {
 
     orderedDimLabelReader @20 :QueryReader;
     # orderedDimLabelReader contains data needed for dense dimension label reads.
+
+    channels @21 :List(QueryChannel);
```

Comment:
We should talk about this... We always have the default channel but we shouldn't serialize it. The default channel aggregates should only be serialized if we have aggregates.